Generated project fails to build during initial `npm install`

Steps to reproduce:

```bash
vue init Akryum/vue-share-components vuejs-blockly
cd vuejs-blockly
npm install
```

Error:

```bash
[...]

> vuejs-blockly@0.0.1 build:browser /Users/Vincent/Code/vuejs-blockly
> cross-env NODE_ENV=production webpack --config config/webpack.config.browser.js --progress --hide-modules

Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
 - configuration.output.path: The provided value "./dist" is not an absolute path!

[...]

npm WARN vuejs-blockly@0.0.1 No repository field.
npm ERR! Darwin 16.5.0
npm ERR! argv "/usr/local/Cellar/node/7.8.0/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v7.8.0
npm ERR! npm  v4.2.0
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! vuejs-blockly@0.0.1 prepublish: `npm run build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the vuejs-blockly@0.0.1 prepublish script 'npm run build'.

[...]
```

--- a/template/config/webpack.config.browser.js
+++ b/template/config/webpack.config.browser.js
@@ -1,13 +1,14 @@
 var webpack = require('webpack')
 var merge = require('webpack-merge')
 var base = require('./webpack.config.base')
+var path = require('path')
 
 var outputFile = '{{ name }}'
 var globalName = '{{ library }}'
 
 module.exports = merge(base, {
   output: {
-    path: './dist',
+    path: path.resolve(__dirname, '../dist'),
     filename: outputFile + '.browser.js',
     library: globalName,
     libraryTarget: 'umd',

--- a/template/config/webpack.config.common.js
+++ b/template/config/webpack.config.common.js
@@ -1,13 +1,14 @@
 var webpack = require('webpack')
 var merge = require('webpack-merge')
 var base = require('./webpack.config.base')
+var path = require('path')
 
 var outputFile = '{{ name }}'
 var globalName = '{{ library }}'
 
 module.exports = merge(base, {
   output: {
-    path: __directory + '/dist',
+    path: path.resolve(__dirname, '../dist'),
     filename: outputFile + '.common.js',
     libraryTarget: 'commonjs2',
   },

--- a/template/config/webpack.config.dev.js
+++ b/template/config/webpack.config.dev.js
@@ -1,12 +1,13 @@
 var merge = require('webpack-merge')
 var base = require('./webpack.config.base')
+var path = require('path')
 
 var outputFile = '{{ name }}'
 var globalName = '{{ library }}'
 
 module.exports = merge(base, {
   output: {
-    path: './dist',
+    path: path.resolve(__dirname, '../dist'),
     filename: outputFile + '.common.js',
     library: globalName,
     libraryTarget: 'umd',

